Products/CMFPlone/tests/testSyndication.py

```diff
@@ -1,25 +1,25 @@
 # -*- coding: utf-8 -*-
-import re
-from Products.CMFCore.utils import getToolByName
 from AccessControl import Unauthorized
 from plone.app.textfield import RichTextValue
-from Products.CMFPlone.tests import PloneTestCase
+from plone.registry.interfaces import IRegistry
+from Products.CMFCore.utils import getToolByName
+from Products.CMFPlone.browser.syndication.adapters import DexterityItem
+from Products.CMFPlone.interfaces.syndication import IFeed
 from Products.CMFPlone.interfaces.syndication import IFeedSettings
 from Products.CMFPlone.interfaces.syndication import ISiteSyndicationSettings
-from plone.registry.interfaces import IRegistry
-from zope.component import getUtility
+from Products.CMFPlone.tests import PloneTestCase
 from zExceptions import NotFound
-from Products.CMFPlone.interfaces.syndication import IFeed
-from Products.CMFPlone.browser.syndication.adapters import DexterityItem
+from zope.component import getUtility
 
+import re
 
-class BaseSyndicationTest(PloneTestCase.PloneTestCase):
 
+class BaseSyndicationTest(PloneTestCase.PloneTestCase):
     def afterSetUp(self):
-        self.syndication = getToolByName(self.portal, 'portal_syndication')
-        self.folder.invokeFactory('Document', 'doc1')
-        self.folder.invokeFactory('Document', 'doc2')
-        self.folder.invokeFactory('File', 'file')
+        self.syndication = getToolByName(self.portal, "portal_syndication")
+        self.folder.invokeFactory("Document", "doc1")
+        self.folder.invokeFactory("Document", "doc2")
+        self.folder.invokeFactory("File", "file")
```

Comment:
Huh? Don't we prefer single quotes?

Reply:
huh, seems I used black with wrong params here. yeah, I would love to go to black defaults, but not now. First 5.2, then code style discussions ;)